### DISCUSSION TITLE
Coral USB Accelerator

### DISCUSSION THREAD

**Discussion by sudo-joe** [blakeblackshear/frigate#3908] [General]
Hello

for testing Frigate, the external Coral USB Accelerator seems easy to me. Is it fully supported from Frigate?

Thank you!

**Comment by NickM-27** [comment]
Can you give a link to the one you're referring to?

USB and PCIe corals are fully supported. The dev board coral however is not well supported.

**Comment by sudo-joe** [comment]
of course

https://coral.ai/products/accelerator/

**Reply by NickM-27** [reply]
Yes that is supported

**Reply by sym0nd0** [reply]
> of course
> 
> https://coral.ai/products/accelerator/

This is the model I use and it works really well.
